src/relay.js

```diff
@@ -249,28 +251,31 @@ export function sequelizeConnection({
       options.where = argsToWhere(args);
       options.required = false;
 
-      if (args.after || args.before) {
-        let cursor = fromCursor(args.after || args.before);
-        let startIndex = Number(cursor.index);
+      let cursor = null;
+      if (args.after || args.before) cursor = fromCursor(args.after || args.before);
 
-        if (startIndex > 0) options.offset = startIndex + 1;
+      if (cursor) {
+        let startIndex = Number(cursor.index);
+        if (startIndex > 0) {
+          options[args.after ? 'offset' : 'limit'] = args.after ? startIndex + 1 : startIndex;
+        }
       }
```

Comment:
There is functional change baked into the style change. Which portion exactly is looking just like a style change?